sessions/recap-project-1/quiz-app/README.md

```diff
@@ -22,6 +22,7 @@ Start by:
 - Create a remote repository and connect it to the local repository.
 - Create the `index.html` and `styles.css` files.
 - Make sure the CSS file is loaded in the HTML document.
+- 🚀 Project Deployment to GitHub Pages is Required: Please adhere to the deployment guidelines outlined in your repository's documentation (refer to ./docs/github-pages.md) for detailed instructions. 
```

Comment:
- `Required` should be `required`
- Please add a Markdown link to the guidelines; this will make it easier for the students to find it.

Reply:
Use the exact path you wrote above, @Klausstille. It is a relative URL, so whichever repo it's in, it should work. Please test, and let me know if you have any issues.

Reply:
You can find examples of this in the README.md, for example.